### PR TITLE
#161256141 Application should deploy on Heroku 

### PR DESCRIPTION
**Pull request by pabishai**
**What does this PR do?**
Enable the application to deploy on Heroku

**Description of Task to be completed?**
Remove proxy from the package.json
Add BASE_URL environmental variable

**How should this be manually tested?**
Go to [https://magnificent6-frontend.herokuapp.com/ ](`https://magnificent6-frontend.herokuapp.com/ `)
Click on Get started
Fill in all the fields in the form
It should return a success message

**What are the relevant pivotal tracker stories?**
[#161256141](https://www.pivotaltracker.com/story/show/161256141)

**Screenshots**

Signup form
screen shot 2018-10-17 at 10 45 38

Success Message
screen shot 2018-10-17 at 10 46 00

### CODE REVIEW

File: .env-sample [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_BASE_URL=heroku frontend url
```
Comment (Bileonaire) [inline .env-sample]: @PAbishai Add a new line at the end

File: src/__tests__/jest/actions/authUser.test.js [hunk 2]
```diff
@@ -15,10 +15,12 @@ import {
   signupError
 } from "../../../redux/actions/authUser";
 
+import config from "../../../config";
+
 const middlewares = [thunk];
 const mockStore = configureMockStore(middlewares);
 let store;
-const signupUrl = "http://127.0.0.1:8000/api/users/signup/";
+const signupUrl = `${config}/users/signup/`;
```
Comment (Bileonaire) [inline src/__tests__/jest/actions/authUser.test.js]: @PAbishai shouldn't it be `${config.BASE_URL}`

Comment (pngg) [reply src/__tests__/jest/actions/authUser.test.js]:
```suggestion
const signupUrl = `${config}/users/signup/`;
```
Besides this small change, I think everything is working as expected.

Comment (kimotho-njoki) [reply src/__tests__/jest/actions/authUser.test.js]: @PAbishai response to @Bileonaire 's question?

Comment (pabishai) [reply src/__tests__/jest/actions/authUser.test.js]: @Bileonaire you are right. Amending that.

File: .env-sample (hunk 1 above)
Comment (pabishai) [reply .env-sample]: Added

**Pull request merged**